src/scripts/dungeons/Dungeon.ts

```diff
@@ -1022,6 +1022,18 @@ dungeonList['Mt. Coronet South'] = new Dungeon('Mt. Coronet South',
     ],
     60500, 201, 20);
 
+dungeonList['Solaceon Ruins'] = new Dungeon('Solaceon Ruins',
+    ['Zubat', 'Geodude', 'Natu', 'Bronzor', 'Hippopotas'],
+    [GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.Lucky_incense],
+    960000,
+    [
+        new DungeonBossPokemon('Unown (E)', 4100000, 30),
+        new DungeonBossPokemon('Unown (L)', 4100000, 30),
+        new DungeonBossPokemon('Unown (N)', 4100000, 30),
+        new DungeonBossPokemon('Unown (O)', 4100000, 30),
```

Comment:
Maybe we could replace the `O` with `U`?

DISCOR
ALPH
~~SOLAC~~E~~O~~N ~~R~~U~~I~~N~~S~~
```suggestion
        new DungeonBossPokemon('Unown (U)', 4100000, 30),
```

Reply:
Could do. But to be consistent, we should then also add it to Alph as well, along with N and a new F.

D I S C O R
~~R~~ U ~~I~~ N ~~S~~   ~~O~~ F   A L P H 
~~S O~~ L A ~~C~~ E ~~O~~ N   ~~R~~ U ~~I~~ N ~~S~~

Reply:
Sounds good 👍

Reply:
Don't know why the O was there tbh. Twas supposed to be A E L N. That's my B.